src/test/resources/calls/implicitcast/implicitcast.cpp

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+using namespace std;
+
+int multiply (int val)
+{
+  return val * 3;
+}
+
+
+int main() {
+  std::cout << multiply(10.0) << '\n';
```

Comment:
Just to understand it correctly: We are not checking whether the call would actually succeed? I am a bit confused how we can decide which calls are ok and which ones are not. So the call would always get resolved if one primitive is used, regardless of which one?

Reply:
If the CallResolver does not find any target to invoke before we create a dummy FunctionDeclaration, we first check if we can reach a FunctionDeclaration by modifying its signature with implicit casts.

I agree with you that we need to check if the language is C/C++, because other languages such as Java do not support this feature.